content/blog/2021/11/camel-k-release-1-7/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Camel K 1.7 release"
+date: 2021-11-17
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K"]
+preview: "What's new in Camel K 1.7"
+---
+
+Today we're happy to announce a new major release for Camel K: [Camel K 1.7 version](https://github.com/apache/camel-k/releases/tag/v1.7.0), quite a big leap from latest 1.6 we released a couple of months ago. Keep reading to discover what's new in Camel K! 
+
+First of all, let's highlight the set of technologies on which Camel K 1.7 is based:
+
+* Apache Camel K Runtime 1.10.0
+* Apache Camel Quarkus 2.4.0
+* Apache Camel 3.12.0
+* Apache Camel Kamelets 0.5.0
+
+Thanks to Apache Camel, Camel Quarkus and Kamelet Catalog contributors for the great effor they've put in those new releases as well.
+
+## Dependencies upgrade
+
+The upgrades we applied in this new release, deserve a dedicate section.
+
+We upgraded `Golang` version to `1.6` which gave us the possibility to **bump several Kubernetes API** and so, be in line with the latest enhancements released in K8S space. We also moved to the latest `maven` version, which now point to `3.8.3`. We took the opportunity to promote `Knative` and `Service Binding` to their first `1.x` stable release and to move `Jolokia` to version `1.7.1`. You can find more details on the [GitHub release page](https://github.com/apache/camel-k/releases/tag/v1.7.0).
```

Comment:
I would suggest not to apply inline code block styling to names like Golang, Knative, Service Binding, Maven, Jolokia.